### ISSUE TITLE
TransactionBuilder doesn't set type of Bundle

### ISSUE THREAD

**Issue by mbaltus** [ewoutkramer/fhir-net-api#367]
When using the TransactionBuilder to setup a Bundle for transactions, the type of the Bundle is not set. I would expect to be able to do something like this, and have it work:

```
            var tb = new TransactionBuilder("myendpoint");
            tb.Create(pat);

            var result = client.Transaction(tb.ToBundle());
```

But since the type is not set, servers respond with an error.
Could this be added to the creator as an extra parameter, or perhaps even be set automatically since you only use the TransactionBuilder for setting up a transaction or a batch?

**Comment by ewoutkramer** [comment]
Yes, that makes total sense.

**Comment by ewoutkramer** [comment]
I have added a (default) parameter to the constructors of TransactionBuilder.  Included in 51896f5
